tests/inventory/hosts.yml

```diff
@@ -0,0 +1,23 @@
+---
+tests_cases:
```

Comment:
Good to see coverage increasing. It would be nice to cover more of the possible variables, but that can be left for a separate change. I'd also like to see some validation of the generated workflows, whether it's yamllint, YAML syntax check, or commit the results to the repo so that changes can be manually inspected.

Reply:
We could in a future PR include something such as https://github.com/mpalmer/action-validator#in-a-github-action

Reply:
That would be a nice improvement.